src/pages/faq/runtime.md

```diff
@@ -96,7 +97,44 @@ add the `tappable` attribute to your element.
  <div tappable (click)="doClick()">I am clickable!</div>
 ```
 
-### Cordova plugins not working in the browser
+## Angular Change Detection
```

Comment:
Should this go as a runtime "error" ?

Reply:
Yeah agreed, maybe @liamdebeasi you can make a Development Issues section and put it under there for now?  Otherwise, looks ready to merge.